src/Network/Network/Az.Network.psd1

```diff
@@ -395,7 +395,13 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'Get-AzExpressRouteCrossConnectionRouteTable', 
                'Get-AzExpressRouteCrossConnectionRouteTableSummary', 
                'Get-AzNatGateway', 'New-AzNatGateway', 'Remove-AzNatGateway', 
-               'Set-AzNatGateway'
```

Comment:
If you're removing InterfaceEndpoint, please also remove `Get-AzInterfaceEndpoint` from this file

Reply:
Already removed Get-AzInterfaceEndpoint in pdf1 file.